From fd410678dc8043f7f5bc5d4379aacc3b4d980556 Mon Sep 17 00:00:00 2001
From: Varsha Maira <vmaira@habana.ai>
Date: Wed, 10 Apr 2024 17:35:43 +0300
Subject: [PATCH] [SW-180536] gaudi3: export sha384_csum api from rom

Export api to compute sha384 from rom to re-use in ppboot.

Change-Id: Id20afd62ed8985d7ee4ed178fb6873a0ac0a14d1
Signed-off-by: Varsha Maira <vmaira@habana.ai>
---

diff --git a/arch/gaudi3/arc/arcboot_rom.txt b/arch/gaudi3/arc/arcboot_rom.txt
index 5dbc0b8..102840c 100644
--- a/arch/gaudi3/arc/arcboot_rom.txt
+++ b/arch/gaudi3/arc/arcboot_rom.txt
@@ -33,3 +33,4 @@
 pcie_phy_ssc_en_set
 pcie_phy_set_sram_ext_ld_done
 pcie_set_pcie_device_vendor_id
+sha384_csum
